hammett/stopwatch/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""The package contains functions to collect handler statistics and further represent them."""
```

Comment:
`The package contains functions to collect handler stats and represent them further.`